LogMonitor/src/LogMonitor/Main.cpp

```diff
@@ -278,6 +279,8 @@ int __cdecl wmain(int argc, WCHAR *argv[])
     }
     else
     {
+        SetConsoleCtrlHandler(ControlHandle, TRUE);

```

Comment:
Why can't this be done before the if statement above? I mean change ControlHandle function to propogate control event and also set g_hStopEvent. And then you don't need ProcessMonitorControlHandle function you added in another file.